File: Cookers/Cookers/ClientApp/src/app/recipe.ts
import { Step } from "./step";
import { Ingredient } from "./ingredient";

export interface Recipe {
    id: number;
    name: string;
    description: string;
    photo: string;
    cookingTime: number;
    peoples: number;
    favorites: number;
    likes: number;
    steps: Step[];
    ingridients: Ingredient[];
}

File: Cookers/Core/Entity/Recipe.cs
using System.Collections.Generic;

namespace Domain.Entity
{
    public class Recipe
    {
        public int Id { get; private set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int CookingTime { get; set; }
        public int PersonCount { get; set; }
        public int Favourites { get; set; }
        public int Likes { get; set; }
        public List<Step> Steps { get; set; }
        public List<Recipe> Recipes { get; set; }
        public List<Ingredient> Ingredients { get; set; }
    }
}
